product_pack/views/product_template_views.xml

```diff
@@ -9,7 +9,7 @@
         <field name="inherit_id" ref="product.product_template_form_view" />
         <field name="arch" type="xml">
             <div name="options" position="inside">
-                <div>
+                <div groups="product_pack.group_product_pack_user">
```

Comment:
Groups should be at view level, not field level. This way, the access problems may arise.

Reply:
I don't agree. As for analytic for example, it allows to 'enable' a feature.

So, if user has access to packs, this div should be visible.

Reply:
That's not what I'm saying. I'm talking only at technical level.

Reply:
So ?

Reply:
I mean that if you protect by record rules some related (m2o, m2m) fields, putting the group that grants the access to them at other level that is not the view one, you will get a permission error when accessed by users without such group.

Reply:
That's not what I mean. The question is that a field is included in the view definition, it's fetched, and then if you don't have access to target model, there will be an error. The only way to avoid that is to protect the entire view with the group, not only a part of it.

Reply:
Done